Mage.Sets/src/mage/cards/u/UlalekFusedAtrocity.java

```diff
@@ -91,7 +91,7 @@ public boolean apply(Game game, Ability source) {
                 .stream()
                 .filter(obj -> !(obj instanceof Spell))
                 .filter(obj -> obj.isControlledBy(source.getControllerId()))
-                .filter(obj -> !obj.getId().equals(source.getSourceId()))
+                .filter(obj -> !obj.getId().equals(source.getId()))
```

Comment:
Are you sure it really filter source ability?

Reply:
Yes, otherwise it's checking the permanent's ID - which isn't what we want. Technically Ulalek can copy other triggered abilities from the same permanent if for instance [[Roaming Throne]] is on the field.